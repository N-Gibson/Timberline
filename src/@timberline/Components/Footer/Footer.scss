footer {
  background-color: #c8ad96;
  padding: 10px;
  height: 100px;
}

.footer_content {
  margin: 5px;
}

.footer_subset {
  display: flex;
  justify-content: center;
}
