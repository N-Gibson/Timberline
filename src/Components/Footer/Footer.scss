footer {
  background-color: #c8ad96;
  padding: 10px;
  height: 10vh;
}

.footer_content {
  margin: 5px;
}
