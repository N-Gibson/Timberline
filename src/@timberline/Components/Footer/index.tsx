import React from 'react'
import PermContactCalendarIcon from '@material-ui/icons/PermContactCalendar'
import CallIcon from '@material-ui/icons/Call'
import EmailIcon from '@material-ui/icons/Email'

import './Footer.scss'

const Footer = () => {
  return (
    <footer>
      <div className="footer_subset">
        <PermContactCalendarIcon />
        <p className="footer_content">Contact Us</p>
      </div>
      <div className="footer_subset">
        <CallIcon />
        <p className="footer_content">(505)-470-5446</p>
      </div>
      <div className="footer_subset">
        <EmailIcon />
        <p className="footer_content">timberlineincsantafe@gmail.com</p>
      </div>
    </footer>
  )
}

export { Footer }
