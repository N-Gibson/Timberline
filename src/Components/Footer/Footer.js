import React from 'react';
import './Footer.scss';

export const Footer = () => {
  return (
    <footer>
      <p className="footer_content">Contact Us</p>
      <p className="footer_content">Phone: (505)-470-5446</p>
      <div className="footer_content">
        <a href="mailto: timberlineinc.steve@gmail.com" >Email: timberlineinc.steve@gmal.com</a>
      </div>
      {/* <p className="footer_content">timberlineinc.steve@gmail.com</p> */}
    </footer>
  )
}
