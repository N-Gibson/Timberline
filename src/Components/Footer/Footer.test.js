import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './Footer';

describe('Footer', () => {
  const wrapper = shallow(<Footer />);

  it('Footer component should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
