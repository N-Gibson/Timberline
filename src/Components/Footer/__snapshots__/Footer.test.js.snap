// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer component should match snapshot 1`] = `
<footer>
  <p
    className="footer_content"
  >
    Contact Us
  </p>
  <p
    className="footer_content"
  >
    Phone: (505)-470-5446
  </p>
  <a
    href="mailto: timberlineinc.steve@gmail.com"
  >
    Email: timberlineinc.steve@gmal.com
  </a>
</footer>
`;
